c++/src/kj/exception.h

```diff
@@ -132,8 +132,15 @@ class Exception {
   friend class ExceptionImpl;
 };
 
+struct CanceledException { };
+  // This exception is thrown to force-unwind a stack in order to immediately cancel whatever that
```

Comment:
Nitpick: The doc comment should only be indented like this if it's inside the `{}` block. Since it isn't, reduce indentation by one level.